### DISCUSSION TITLE
Queries work fine on local but I'm somehow getting "Could not find driver connection: , SQL: ..." error in production

### DISCUSSION THREAD

**Discussion by Kenneth-Inkum** [squirephp/squire#88] [Q&A]
The package works fine on my local machine, but I can't make queries in production. The application keeps looking for the models in the database. I need help, please.

![image](https://github.com/user-attachments/assets/ff404e73-3c7c-4b25-9e81-dfa374646383)

**Comment by ShaungBhone** [comment]
Same here how can I fix that?
